packager/media/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 Google Inc. All rights reserved.
```

Comment:
what is right business name for Google? LLC or Inc? as I see both in different files

Reply:
It should be LLC.  We did a mass-rename operation in this branch not long ago, and this slipped back in.  A minor copy-paste bug.  :-)

Reply:
Fixed!